e2e/suites/extensions/ext-info-drawer.test.ts

```diff
@@ -0,0 +1,176 @@
+
+import { LoginPage, LogoutPage, BrowsingPage } from '../../pages/pages';
+import { InfoDrawer } from './../../components/info-drawer/info-drawer';
+import { RepoClient } from '../../utilities/repo-client/repo-client';
+import { SIDEBAR_LABELS, EXTENSIBILITY_CONFIGS } from '../../configs';
+import { Utils } from '../../utilities/utils';
+
+xit('');
+
+describe('Extensions - Info Drawer', () => {
+    const username = `user-${Utils.random()}`;
+
+    const file = `file-${Utils.random()}.txt`;
+    let fileId;
+
+    const properties_tab = {
+        order: 1,
+        title: 'MY PROPERTIES'
+    };
+
+    const custom_tab = {
+        order: 2,
+        icon: 'mood',
+        title: 'MY CUSTOM TITLE',
+        component: 'app.toolbar.toggleFavorite'
+    };
+
+    const no_title_tab = {
+        order: 3,
+        icon: 'check_circle',
+        title: ''
+    };
+
+    const comments_tab = {
+        title: 'COMMENTS'
+    };
+
+    const apis = {
+        admin: new RepoClient(),
+        user: new RepoClient(username, username)
+    };
+
+    const infoDrawer = new InfoDrawer();
+
+    const loginPage = new LoginPage();
+    const logoutPage = new LogoutPage();
+    const page = new BrowsingPage();
+
+    beforeAll(async (done) => {
+        await apis.admin.people.createUser({ username });
+        fileId = (await apis.user.nodes.createFile(file)).entry.id;
+        done();
+    });
+
+    afterAll(async (done) => {
+        await apis.user.nodes.deleteNodeById(fileId);
+        done();
+    });
+
+    describe('', () => {
+        beforeAll(async (done) => {
+            await loginPage.load();
+            await Utils.setSessionStorageFromConfig('"aca.extension.config"', EXTENSIBILITY_CONFIGS.INFO_DRAWER);
+            await loginPage.loginWith(username);
+            done();
+        });
+
+        beforeEach(async (done) => {
+            await page.sidenav.navigateToLinkByLabel(SIDEBAR_LABELS.PERSONAL_FILES);
+            await page.dataTable.waitForHeader();
+            await page.dataTable.clearSelection();
+            done();
+        });
+
+        afterAll(async (done) => {
+            await logoutPage.load();
+            done();
+        });
+
+        it('Add a new tab with icon and title - [C284646]', async () => {
+            await page.dataTable.selectItem(file);
+            await page.toolbar.actions.getButtonByTitleAttribute('View details').click();
+            await infoDrawer.waitForInfoDrawerToOpen();
+
+            expect(await infoDrawer.isTabPresent(custom_tab.title)).toBe(true, `${custom_tab.title} tab is not present`);
+            expect(await infoDrawer.getTabTitle(custom_tab.order)).toEqual(`${custom_tab.icon}\n${custom_tab.title}\n`);
+        });
+
+        it('Remove existing tab - [C284647]', async () => {
+            await page.dataTable.selectItem(file);
+            await page.toolbar.actions.getButtonByTitleAttribute('View details').click();
+            await infoDrawer.waitForInfoDrawerToOpen();
```

Comment:
Better use a method with these 3 lines

Reply:
good point! I have created a separate task for this: ACA-1709 since I have other places in other tests that could benefit from this.